cfgov/cfgov/settings/base.py

```diff
@@ -655,7 +655,9 @@
     'FINANCIAL_COACHING': {},
 
     # Teacher's Digital Platform
-    'TDP_RELEASE': {},
+    'TDP_RELEASE': {
+		'site': 'beta.consumerfinance.gov:443',
```

Comment:
Tabs seem weird here. Didn't you say something about redoing your vim plugins on Monday morning? :smile:

Reply:
Yes. Yes, I did.